### DISCUSSION TITLE
Show more information in the request detail graph

### DISCUSSION THREAD

**Discussion by MoritzSalein** [ludeknovy/jtl-reporter#147] [Ideas]
When I go to the detailed reqeust metric, there are only filters for 99%, 95%, 90%, througput, error rate and threads. However, my customer also wants to see latency (avg), connect time (avg) and response time (avg).
Is it possible to add these filters?
For me the best presentation of these values is a bar chart. What do you think about this?

**Comment by ludeknovy** [comment]
Yes, they can be added!

**Reply by ludeknovy** [reply]
@MoritzSalein 

https://github.com/ludeknovy/jtl-reporter/issues/153
implemented. You can check it out in "latest", you need both fe and be latest tag.


And the new metrics are only available for newly added reports.

**Reply by ludeknovy** [reply]
or it is available in v4.3.0 as well.

**Comment by MoritzSalein** [comment]
In the todays v.4.3.0 version, I can't see any change in the detailed request metric. I'll try it in the lastest version.

**Reply by ludeknovy** [reply]
To have the latency and connect time available, you need to upload a new report.

**Reply by ludeknovy** [reply]
It should look like this.

![Screenshot 2022-02-28 at 15 49 35](https://user-images.githubusercontent.com/13610612/156003618-c75dfeba-5fa2-40a5-bf98-308a860ee2de.png)

**Comment by MoritzSalein** [comment]
Sorry, I think I'm a little bit stupid. I stoped my old environment with docker-compose down, load the current version from git and set up the system wir docker-compse up -d. After completion I see an empty system, create an account und add new Projekt, Scenario and the test runs by imporing the jtl-Data. But at the end, I didn't got the filter of the additional values. 
What is going wrong with me?

**Reply by ludeknovy** [reply]
Hi, you are not stupid ;)

What version do you use? You should be using v4.3.0 for the backend and frontend. Does your jtl files include `Connect` and `Latency` columns?

Btw. you dont need to clone the whole repo over and over again, its enough if you update the docker-compose file.

**Comment by MoritzSalein** [comment]
In my docker-composer files are these lines included:
`  fe:
    image: novyl/jtl-reporter-fe:v4.3.0
...
  be:
    image: novyl/jtl-reporter-be:v4.3.0`

The cloumn in the jtl has this entries:
timeStamp,elapsed,label,responseCode,responseMessage,threadName,dataType,success,failureMessage,bytes,sentBytes,grpThreads,allThreads,URL,Latency,Hostname,IdleTime,Connect

Example of one entry is:
1645712500109,61,webshop: GET /index/,200,OK,10.100.1.239-Thread Group 1-6,text,true,,3230,526,12,12,https://testsystem.xx/index/,61,ip-10-100-1-239.eu-central-1.compute.internal,0,31

**Reply by ludeknovy** [reply]
hmm, that all looks correct :/

Will need more time to investigate it.

**Reply by MoritzSalein** [reply]
It is curious. But after stopping and restarting the JtlReporter Docker  container, I got all new values. It seems a problem of my browser caching.
![2022-03-03 15_09_20-JtlReporter](https://user-images.githubusercontent.com/83296020/156581098-16b21e49-ed49-4533-9beb-1096bd578781.png)

**Reply by ludeknovy** [reply]
Good! I just realized I forgot to add the requested "response time (avg)"

**Comment by MoritzSalein** [comment]
It's possible to add the "Response Time [avg]"?

**Reply by ludeknovy** [reply]
Sure! Sorry, my bad - I forgot it.

**Reply by MoritzSalein** [reply]
No prob.

**Reply by ludeknovy** [reply]
Implemented and merged into master branch.
